app/services/optout/OptOutService.scala

```diff
@@ -157,43 +135,54 @@ class OptOutService @Inject()(itsaStatusUpdateConnector: ITSAStatusUpdateConnect
 
     def findAnyFailOrFirstSuccess(responses: Seq[OptOutUpdateResponse]): OptOutUpdateResponse = {
       responses.find {
-          case _: OptOutUpdateResponseFailure => true
-          case _ => false
+        case _: OptOutUpdateResponseFailure => true
+        case _ => false
       }.getOrElse(responses.head)
     }
 
     val result = for {
       intentTaxYear <- OptionT(fetchSavedIntent())
       yearsToUpdate <- OptionT(Future.successful(Option(optOutProposition.optOutYearsToUpdate(intentTaxYear))))
-      responsesSeqOfFutures  <- OptionT(Future.successful(Option(makeUpdateCalls(yearsToUpdate))))
+      responsesSeqOfFutures <- OptionT(Future.successful(Option(makeUpdateCalls(yearsToUpdate))))
       responsesSeq <- OptionT(Future.sequence(responsesSeqOfFutures).map(v => Option(v)))
       finalResponse <- OptionT(Future.successful(Option(findAnyFailOrFirstSuccess(responsesSeq))))
     } yield finalResponse
 
     result.getOrElse(OptOutUpdateResponseFailure.defaultFailure())
   }
 
+  def nextUpdatesPageOptOutViewModel(proposition: OptOutProposition)(implicit user: MtdItUser[_], hc: HeaderCarrier, ec: ExecutionContext): Option[OptOutViewModel] = {
```

Comment:
unused params, can be removed: (implicit user: MtdItUser[_], hc: HeaderCarrier, ec: ExecutionContext)